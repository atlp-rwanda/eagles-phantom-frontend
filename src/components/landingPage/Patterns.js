import React,{ Component }  from 'react';
import Pattern from './PatternsCard';
import '../../scss/homeContent.scss';

export default class Patterns extends Component {
    render(){
    return(
        <div className="patterns_with">
        <h3 className="patterns_title">Our Parteners</h3>
        <div className="patterns_card">
        <Pattern
            description="RFTC"
        />
        <Pattern
            description="Royal"
        />
        <Pattern
            description="AC Group ltd"
        />
        </div>
        </div>
    )
    }
}