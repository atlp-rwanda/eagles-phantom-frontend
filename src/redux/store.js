import {createStore, applyMiddleware } from 'redux';
import { composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import promise from 'redux-promise-middleware';
import allReducers from './reducers/';

// const rootReducer = combineReducers({
//   count: counterReducer,
// });

// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const middleware = applyMiddleware(thunk, promise)
const store = createStore(allReducers, {}, composeWithDevTools(middleware));

export default store;
