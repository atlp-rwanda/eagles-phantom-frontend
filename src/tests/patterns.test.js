import React from 'react';
import { Provider } from "react-redux";
import store from "../redux/store.js";
import { render } from '@testing-library/react';
import Patterns from '../components/landingPage/Patterns';

const patterns=()=>{
    return render(
       <Provider store={store}>
       <Patterns />
       </Provider>
    )
}
describe("Patterns",()=>{
    it("shoould render the component",()=>{
        const { asFragment }=patterns();
        expect(asFragment(<Patterns />)).toMatchSnapshot()
    });
});

