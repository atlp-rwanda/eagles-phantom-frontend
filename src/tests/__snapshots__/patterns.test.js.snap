// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patterns shoould render the component 1`] = `
<DocumentFragment>
  <div
    class="patterns_with"
  >
    <h3
      class="patterns_title"
    >
      Our Parteners
    </h3>
    <div
      class="patterns_card"
    >
      <div
        class="patterns"
      >
        <p>
          RFTC
        </p>
      </div>
      <div
        class="patterns"
      >
        <p>
          Royal
        </p>
      </div>
      <div
        class="patterns"
      >
        <p>
          AC Group ltd
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
